src/server/game/Entities/Object/Object.h

```diff
@@ -540,6 +540,14 @@ struct Position
 
     float GetAngle(const Position* pos) const;
     [[nodiscard]] float GetAngle(float x, float y) const;
+    [[nodiscard]] float GetAbsoluteAngle(float x, float y) const
+    {
+        float dx = x - m_positionX;
+        float dy = y - m_positionY;
+        return NormalizeOrientation(std::atan2(dy, dx));
```

Comment:
why not just:

```cpp
return NormalizeOrientation(std::atan2(y - m_positionY, x - m_positionX));
```

we don't need to create 2 more variables, do we?

Reply:
Probably just for easier readability

Reply:
@Kitzunu then why don't just declare them  as`inline` ?

Reply:
I dont know what that is O.O

Reply:
also doing it as you said, wouldnt that mean it is not float anymore?

Reply:
`inline float dx = x - m_positionX;`

Reply:
@Kitzunu 

> C++ provides an inline functions to reduce the function call overhead. Inline function is a function that is expanded in line when it is called. When the inline function is called whole code of the inline function gets inserted or substituted at the point of inline function call. This substitution is performed by the C++ compiler at compile time. Inline function may increase efficiency if it is small.

Since C++17 we also got `inline` variables, based on the same principle. You need to extract a variable/function for better readability, but you don't really want to create a variable in memory.

Reply:
ah ok, I can add that

Reply:
@Kitzunu yes, please test it again after changing it just to double check

Reply:
absolutely